extensions/control-plane/transfer/transfer-data-plane/src/main/java/org/eclipse/edc/connector/transfer/dataplane/flow/ConsumerPullTransferDataFlowController.java

```diff
@@ -39,36 +40,52 @@ public class ConsumerPullTransferDataFlowController implements DataFlowControlle
     private final DataPlaneSelectorService selectorService;
     private final ConsumerPullDataPlaneProxyResolver resolver;
 
+    private final Set<String> transferTypes = Set.of("%s-%s".formatted("Http", PULL));
```

Comment:
currently the http type is `HttpData`, not `Http`. I know that I wrote that code (:smiling_face_with_tear: ), but the `Push` dataflowcontroller will create `HttpData-PUSH` transer types, so I think this should be consistent.

Furthermore, this can be a string instead of a single entry Set

Reply:
Ok so i will change to `HttpData-PULL` ok?

Reply:
This is a set so we can reuse it for `transferTypesFor` method

Reply:
@ndr-brt  renamed